code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1289,6 +1287,38 @@
 	color = "#FFFFFF" // white
 	random_color_list = list("#FFFFFF") //doesn't actually change appearance at all
 
+ /* used by crayons, can't color living things but still used for stuff like food recipes */
```

Comment:
what the frick is this

Reply:
coby you can't just say that without context i am very dumb and do not understand what you are complaining about

Reply:
everything below this

Reply:
it's the same as color powder but the procs for colouring mobs return, so you can't use it to turn red or black or whatever by itself

Reply:
did you test this? pretty sure you get icky side effects with just flat out returning life and reaction.

Reply:
fairly sure i tested and worked, with drinking it not working but throwing it at tiles painting them will check again. what icky side effects?